.env.example

```diff
@@ -0,0 +1,4 @@
+GATSBY_ALGOLIA_APP_ID=
```

Comment:
If I try to run `yarn build`, I get an error. Is this because I don't have any of the env variables? If so, we probably want to make sure users are able to create a finished build even if users don't have the required api keys. 

![image](https://user-images.githubusercontent.com/43496356/150869259-2390cbb7-11b0-4a62-a444-a0d36bc15b4a.png)

Reply:
Sure, there is a skip indexing option as well. We can set that to true if no variables are provided.